packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -84,10 +88,47 @@ export interface ITreeConfigurationOptions {
 	 * bit slower.
 	 */
 	enableSchemaValidation?: boolean;
+
+	/**
+	 * If `true`, constructing the configuration will error if any valid input data could parse ambiguously
+	 * (type of the node could fail to be inferred from content, thus requiring use of {@link Unhydrated} nodes to disambiguate).
+	 *
+	 * @defaultValue `false`.
+	 *
+	 * @remarks
+	 * When this is true, it ensures that the compile time type safety for data when constructing nodes is sufficient to ensure than the runtime behavior will not give node data ambiguity errors.
+	 *
+	 * This ensures that the canonical JSON like representation of the node of any union in the tree is lossless and unambiguous.
+	 * This canonical JSON like representation consists of arrays, plain old JavaScript objects with string keys, booleans, numbers (excluding NaN, -0 and infinities), strings, null and IFluidHandles.
+	 * Assuming any IFluidHandles get special handling, its JSON compatible, as well as compatible with the node creation APIs (such as schema class constructors).
```

Comment:
The "special handling" here could link to the IFluidSerializer (is that what it's called?) type or something, so the reader knows what we're alluding to.

Reply:
IFluidSerializer is alpha+legacy, so we can't/shouldn't reference it from the public API . Right now the only thing you can do is something like provide a custom callback to JSON.stringify (or your own custom clone logic) which uses isFluidHandle.  Eventually if we add some APIs to import/export JSON compatible data (like those proposed in https://github.com/microsoft/FluidFramework/pull/22022 ) this could reference them.